net/http/src/THttpLongPollEngine.cxx

```diff
@@ -44,16 +54,78 @@ void THttpLongPollEngine::ClearHandle()
    if (fPoll) {
       fPoll->Set404();
       fPoll->NotifyCondition();
-      fPoll = nullptr;
+      fPoll.reset();
+   }
+}
+
+//////////////////////////////////////////////////////////////////////////
+/// Create raw buffer which should be send as reply
+/// For the raw mode all information must be send via binary response
+
+std::string THttpLongPollEngine::MakeBuffer(const void *buf, int len, const char *hdr)
+{
+   std::string res;
+
+   if (!fRaw) {
+      res.resize(len);
+      memcpy((void *) res.data(), buf, len);
```

Comment:
Just because this seems to be a pattern: the non-fragile version (memory-wise) that is equally performant is:
```
res.reserve(len);
std::copy(buf, buf + len, res.begin());
```
I'd appreciate if you could replace the pattern `resize()` / `memcpy()` but that pattern.